### ISSUE TITLE
Adventure map, hero level up window, number of known secondary skills, text order and size

### ISSUE THREAD

**Issue by LeHerosInconnu** [ihhub/fheroes2#1595]
The text corresponding to the number of secondary skills the hero knows and can learn is displayed in the wrong order "8/2" instead of "2/8", "two out of eight" instead of "eight out of two".
Also, the size of the font used is far too large, the same of the font used for the text in the graphics of the secondary skills would be more than sufficient.

In fheroes2:

![Number secondary skills, text order and size 01](https://user-images.githubusercontent.com/43583994/92490193-01edde80-f1f1-11ea-9967-d46131e13dcf.png)

**Comment by ihhub** [comment]
Hi @LeHerosInconnu , from what we know most of players don't know what these numbers are. We might replace this text by something more meaningful.

**Comment by LeHerosInconnu** [comment]
@ihhub.

To begin with, an information window should be displayed when the player right clicks on the hero button area and the "2/8" text area.

For my "Project Ironfist" mod project, I made some buttons to replace the "Next Hero" button by a "HERO" button a bit more understandable for the user.

![Chest hero choice window 03](https://user-images.githubusercontent.com/43583994/92504334-d247d200-f202-11ea-907d-5d25d12db5b1.png)

**Comment by Branikolog** [comment]
> an information window should be displayed when the player right clicks on the hero button area

Agreed. Nevertheless, players are confused only the first time they see this button... The next time they use it freely with pleasure.
The only thing left is to make this window and buttons in it look balanced and user-firendly.

**Comment by LeHerosInconnu** [comment]
@Branikolog.

> The only thing left is to make this window and buttons in it look balanced and user-firendly.

Yes, this window needs a bit of work, the layout of the elements lacks a bit of spacing.

**Comment by ihhub** [comment]
Hi @LeHerosInconnu and @Branikolog , I swapped numbers in this window as eveyone agrees on this. I'm moving this issue for 1.0 release as we need to have a proper solution for this button.
